bin/dev

```diff
@@ -1,8 +1,10 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
 
-if ! gem list foreman -i --silent; then
-  echo "Installing foreman..."
-  gem install foreman
-fi
+PORT="${PORT:-3000}"
+export PORT
 
-exec foreman start -f Procfile.dev "$@"
+if command -v overmind &> /dev/null; then
+  overmind start -f Procfile.dev "$@"
+else
+  foreman start -f Procfile.dev "$@"
```

Comment:
@olimart In this conditional, it's possible the person doesn't have foreman either. Shall we keep the "gem install" block that you removed above but put it down here within this branch of the conditional?

Reply:
we should indeed. I've added it back. thanks